test/framework/docker.go

```diff
@@ -85,7 +85,7 @@ func (d *Docker) ClusterStateValidations() []clusterf.StateValidation {
 }
 
 // WithKubeVersionAndOS returns a cluster config filler that sets the cluster kube version.
-func (d *Docker) WithKubeVersionAndOS(kubeVersion anywherev1.KubernetesVersion, _ OS, _ *releasev1.EksARelease, _ ...bool) api.ClusterConfigFiller {
+func (d *Docker) WithKubeVersionAndOS(kubeVersion anywherev1.KubernetesVersion, _ OS, _ *releasev1.EksARelease, _ ...string) api.ClusterConfigFiller {
```

Comment:
```suggestion
func (d *Docker) WithKubeVersionAndOS(kubeVersion anywherev1.KubernetesVersion, _ OS, _ *releasev1.EksARelease) api.ClusterConfigFiller {
```

Reply:
Same as above - I can't remove the unused arg since I need to have the 4th argument of type `...string` in those methods to satisfy the provider interface.